addons/sourcemod/translations/superzombiefortress.phrases.txt

```diff
@@ -664,6 +664,6 @@
 	
 	"Menu_ClassesInfectedSpecialJockey"
 	{
-		"en"		"The Jockey creeps around, finds survivor head to mock around with it.\n- Using rage allows jockey to pounce into survivors head, but range not as long as hunter\n- Can walk into survivors head and play with it.\nCan't attack, but deals small damage and can mostly control movement while sitting on survivor."
+		"en"		"The Jockey creeps around, eager to lead survivors to their demise.\n- Using rage will perform a short leap, latching onto a survivor's head when making physical contact.\n- Mostly controls a survivor's movement and deals damage while on top of them."
```

Comment:
Would've update all other translations using whatever translator tools to match this, otherwise it may be stuck misleading for a long while.